kitsune/search/urls.py

```diff
@@ -4,8 +4,7 @@
 from kitsune.search.v2 import views as v2_views
 
 urlpatterns = [
-    url(r"^$", views.simple_search, name="search"),
+    url(r"^$", v2_views.simple_search, name="search.v2"),
```

Comment:
My initial thought was to keep the view name as `name="search"` to avoid having to update it elsewhere, but this works